lib/mcb/commands/sites/add_ttwa_or_london_borough.rb

```diff
@@ -0,0 +1,23 @@
+name 'travel_to_work_area_or_london_borough'
+summary "Batch add travel to work area or London Borough if located in London to Sites in the current cycle."
+
+instance_eval(&MCB.remote_connect_options)
+
+run do |opts, _args, _cmd|
+  MCB.init_rails(opts)
+
+current_recruitment_cycle
+  Site.joins(:provider).where('provider.recruitment_cycle_id = ?', RecruitmentCycle.current_recruitment_cycle.id).each do |site|
```

Comment:
- We may want to have an option to only backfill Sites that have no travel to work area. If the back fill fails then you may want to resume from the last successful update, otherwise you eat into your allowance.
- Do we need to check API rate limiting and space out the requests?
- At some point we will need to update the travel to work info when a provider updates their site location. May find we need to extract a lot of this logic to a shared service.

Reply:
For the above i've: